ueimporter/main.py

```diff
@@ -405,6 +413,24 @@ def main():
     if not config.pretend and not verify_plastic_repo_state(config, logger):
         return 1
 
+    if not path_util.is_directory_on_case_sensitive_filesystem(
+            config.plastic_repo.workspace_root):
+        logger.log_warning('Warning: Case insensitive filesystem detected.\n'
+                           'ueimporter have no way of correctly replicating'
```

Comment:
ueimporter *has* no way

Reply:
Thank you, my english is ever improving :)